modules/programs/xplr.nix

```diff
@@ -35,15 +52,20 @@ in {
     package = mkPackageOption pkgs "xplr" { };
 
     plugins = mkOption {
-      type = with types; nullOr (listOf (either package str));
-      default = [ ];
-      defaultText = literalExpression "[]";
+      type = with types; nullOr (attrsOf (either package str));
+      default = { };
+      defaultText = literalExpression "{ }";
       description = ''
-        Plugins to be added to your configuration file.
+        An attribute set of plugin paths to be added to the [package.path]<https://www.lua.org/manual/5.4/manual.html#pdf-package.path> of the {file}`~/config/xplr/init.lua` configuration file.
 
-        Must be a package, an absolute plugin path, or string to be recognized
-        by xplr. Paths will be relative to
-        {file}`$XDG_CONFIG_HOME/xplr/init.lua` unless they are absolute.
+        Must be a package or string representing the plugin directory's path. 
+        If the path is not absolute, it will be relative to {file}`$XDG_CONFIG_HOME/xplr/init.lua`.
+      '';
+      example = literalExpression ''
+        {
+          wl-clipboard = inputs."wl-clipboard.xplr";
```

Comment:
I'd use fetchGit here, for people not using git.

Reply:
Agreed, although I would suggest `fetchFromGitHub`. If you search for that function you'll find already existing examples.

Reply:
Thank you. Resolved by latest force push.